src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/custom.py

```diff
@@ -129,7 +130,24 @@ def _handle_global_configuration():
         global_config.set('logging', 'enable_log_file', 'yes' if enable_file_logging else 'no')
         set_global_config(global_config)
 
-def handle_configure():
+def handle_configure(setting=None, value=None):
+    '''
+    configure common settings
+    :param str setting: configuration variable, e.g. resource-group, appservice/webapp, compute/vm
+    :param str value:   configuration variable value
+    '''
+    if setting or value:
+        if bool(setting) != bool(value):
```

Comment:
supposed to verify both values are `true`. Let me get it clearer by using `if not setting or not value:`